### ISSUE TITLE
Update minmatch/dependencies who pull in minimatch to avoid  RegExp DoS issue?

### ISSUE THREAD

**Issue by yaananth** [gulpjs/gulp#1716]
```
npm WARN deprecated graceful-fs@3.0.8: graceful-fs v3.0.0 and before will fail o
n node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible
. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN deprecated minimatch@2.0.10: **Please update to minimatch 3.0.2 or higher
 to avoid a RegExp DoS issue**
npm WARN deprecated minimatch@0.2.14: **Please update to minimatch 3.0.2 or higher
 to avoid a RegExp DoS issue**
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade
 to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail o
n node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible
```

**Comment by phated** [comment]
No. This is a duplicate issue that's already been answered.

**Comment by yaananth** [comment]
could you link the duplicate?

**Comment by Romanx** [comment]
https://github.com/gulpjs/gulp/issues/1571

**Comment by yaananth** [comment]
awesome thanks! @Romanx

**Comment by brunogabriel902** [comment]
Hi, how to fix problem? npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

**Comment by ChenPHPER** [comment]
hello, i had these problem too, how can i fix it?
